contrib/pipelines/core/store_log.go

```diff
@@ -18,22 +18,39 @@
 package core
 
 import (
+	"fmt"
+
 	"github.com/skydive-project/skydive/flow"
 	"github.com/skydive-project/skydive/logging"
 )
 
 type storeLog struct {
+	pipeline *Pipeline
 }
 
 // SetPipeline setup
 func (s *storeLog) SetPipeline(pipeline *Pipeline) {
+	s.pipeline = pipeline
 }
 
 // StoreFlows store flows in memory, before being written to the object store
 func (s *storeLog) StoreFlows(flows []*flow.Flow) error {
-	for _, fl := range flows {
-		logging.GetLogger().Infof("store flow: %v", fl)
+	transformed := s.pipeline.Transformer.Transform(flows)
+
+	encoded, err := s.pipeline.Encoder.Encode(transformed)
+	if err != nil {
+		logging.GetLogger().Error("Failed to encode object: ", err)
+		return err
+	}
+
+	compressed, err := s.pipeline.Compressor.Compress(encoded)
+	if err != nil {
+		logging.GetLogger().Error("Failed to compress object: ", err)
+		return err
 	}
+
+	fmt.Printf("%s\n", compressed)
```

Comment:
shouldn't be a log here instead a printf ?

Reply:
the name `log` maybe incorrect maybe `stdout` is a better name - let me know? (the idea is that I can get a CSV of JSON file outputed here)

Reply:
indeed :) I thought it was about log. Yes stdout is fine

Reply:
ok